setup/updating.md

```diff
@@ -37,12 +37,20 @@ Updating Coder is a two-step process:
     ```bash
     helm repo update
     ```
+    
+2. Export the current helm chart values into a file:
```

Comment:
I suggest numbering these with 1. 1. 1. so that we can get autonumbering. Also the above text ("Updating Coder is a two-step process") is now inaccurate :wink:

Perhaps we can rephrase to avoid mentioning a number of steps, for future updates, maybe: "To update Coder, follow these steps:" (I am sure @khorne3 can phrase it better though)

Reply:
Agreed with the wording. I almost did the `1.` thing instead actually out of habit but saw the list was actually numbered so I figured I'd keep it, haha. Updating it though because I prefer it too 😋